spring-cloud-deployer-resource-maven/src/main/java/org/springframework/cloud/deployer/resource/maven/MavenProperties.java

```diff
@@ -41,10 +43,13 @@
 	 */
 	private String localRepository = DEFAULT_LOCAL_REPO;
 
+	private static final String DEFAULT_REMOTE_REPO_KEY = "default" + new Random().nextInt();
```

Comment:
Using `random` value here in order not to collide with any other user provided alias with the key `default`.